x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/DocsV3Support.java

```diff
@@ -310,7 +310,7 @@ private String getLink(String key) {
         // Various other remaining old asciidoc links
         // <<match-field-params,match query parameters>>
         return switch (parts[0]) {
-            case "match-field-params" -> makeLink(key, "", "/reference/query-languages/query-dsl-match-query.md");
+            case "match-field-params" -> makeLink(key, "", "/reference/query-languages/query-dsl/query-dsl-match-query.md");
```

Comment:
This is a loose end from the docs restructuring PR that merged yesterday. It should have been in that PR.